tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/pages/dashboard/.content.xml

```diff
@@ -16,16 +16,16 @@
           </container_1_11>
           <container_1_1 jcr:primaryType= "nt:unstructured" sling:resourceType= "/apps/typerefinery/components/structure/container" smColSize= "24" mdColSize= "16" lgColSize= "12">
```

Comment:
where are the functional tests for this?

Reply:
I think functional test cases are handled in different PR